resources/fileTemplates/includes/MD File Description.md.ft

```diff
@@ -0,0 +1,8 @@
+<!---
+    You can describe your module here.
+    We recommend that you add the following information:
+     - implementation details: why and how to use the module, preferably with some example scenarios
+     - any dependencies (usually other modules but could be any important dependencies, libraries, etc)
+     - extension points, APIs, plug-ins, etc
+     - any introduced events
+-->
```

Comment:
Please, add an empty line at the end of the file

Reply:
+ add appropriate description to the resources/fileTemplates/includes/MD File Description.md.html

Reply:
Done.